example/csv/src/test/java/org/apache/calcite/test/CsvTest.java

```diff
@@ -297,6 +297,16 @@ private static StringBuilder escapeString(StringBuilder buf, String s) {
         .returns("EMPNO=130; GENDER=F; NAME=Alice").ok();
   }
 
+  /** Filter that can be slightly handled by CsvFilterableTable. */
+  @Test public void testFilterableWhere3() throws SQLException {
+    final String sql = "select empno, gender, name from EMPS\n"
+            + " where gender = 'F' or empno > 125";
+    sql("filterable-model", sql)
```

Comment:
Why we update our code with `AND` while the test case sql is with `OR`, i kind of confused.

Reply:
> Why we update our code with `AND` while the test case sql is with `OR`, i kind of confused.

Got it. Correct it with "where gender <> 'M' and empno > 125".